app/src/main/java/com/ayush/anonytweet/DashBoard.java

```diff
@@ -7,6 +7,7 @@
 import android.os.Build;
 import android.os.Bundle;
 import android.service.notification.StatusBarNotification;
+import android.support.annotation.LayoutRes;
```

Comment:
You should consistently use either the `android.support.annotation` package or the `...androidresource` package.
Why mix the two? It might be easier to just use android packages, because then no CF dependency needs to be added.
This comment applies to all other files as well.

Reply:
Fixed.